Assets/Scripts/Game/Questing/Actions/DialogLink.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Project:         Daggerfall Tools For Unity
+// Project:         Daggerfall Tools For Unity
```

Comment:
Maybe we could add new patterns that reflect the use case better (ex: "remove dialogue for location (?<aSite>\w+) person (?<anNPC>\w+) item (?<anItem>\w+)").

Reply:
Yes, but it should put in another PR I suppose. I should document in this one what "dialog links" really means though.